docs/source/releases/1.16.3/964-allow-running-tiny_sectors_geostationarysh-script-from-multiple-repos.yaml

```diff
@@ -0,0 +1,176 @@
+        for each specific dataset, with null and 70 sat zen cutoff.  This is helpful
+        for evaluating what data actually exists within a given dataset.
+    * OPTIONAL marker, no comparison to correct imagery
+
+      * geoips test sector --overlay commands for every tiny sector defined in the
+        current repo, just spits out the overlay image output for each sector.
+
+    * REQUIRED marker, comparison against correct imagery checked into the repo
+
+      * tiny_sectors_geostationary.sh call for each tiny sector defined within the
+        current repo.  This includes both non-dataset-specific test sectors, as
+        well as dataset-specific test sectors.  Plugin repos should only have
+        dataset-specific test sectors.
+  files:
+    deleted:
+    - ''
+    moved:
+    - ''
+    added:
+    - 'tests/integration_tests/test_tiny_sectors_geoips.py'
+    modified:
+    - 'tests/integration_tests/test_integration.py'
+  related-issue:
+    internal: GEOIPS#964
+  date:
+    start: 2025-10-09
+    finish: 2025-10-09
+- title: 'Pass reponame into test_geostationary_sectors.sh so we can compare test outputs in the correct repo'
+  description: |
+    Previously hard coded to check test outputs in geoips/tests/outputs, update to
+    pass in repo_name so we compare test outputs in repo_name/tests/outputs.
+
+    Update test_integration.py to pass in geoips reponame, as well as test_geostationary_sectors.sh
+    to use repo_name.
+
+    Also replace asterisk in any log filename with - in interactive pytests.
+  files:
+    deleted:
+    - ''
+    moved:
+    - ''
+    added:
+    - ''
+    modified:
+    - 'tests/integration_tests/tiny_sectors/tiny_sectors_geostationary.sh'
+    - 'tests/integration_tests/test_integration.py'
+  related-issue:
+    internal: GEOIPS#964
+  date:
+    start: 2025-10-07
+    finish: 2025-10-08
+- title: 'Add product-name-base argument to global_terminator_satzen script'
+  description: |
+    Pass product-name-base argument to global_terminator_satzen script, and use it
+    to set an environment variable that is then used in the satzen.yaml procflow
+    config to define the requested products for a given dataset.
+
+    Allows specifying additional Test-Infrared products for use with datasets that may
+    not have the standard Infrared channel.  This was used to create
+    Test-Infrared-ABI-B13-Day-Night, -Day-Only, Night-Only products to allow running
+    Infrared low res tiny sectors for the geocolor goes-18 test dataset, which did
+    NOT have B14BT variable available.
+  files:
+    deleted:
+    - ''
+    moved:
+    - ''
+    added:
+    - ''
+    modified:
+    - 'tests/example_scripts/global_terminator_satzen.sh'
+    - 'tests/example_scripts/satzen.yaml'
+  related-issue:
+    internal: GEOIPS#964
+  date:
+    start: 2025-10-07
+    finish: 2025-10-08
+- title: 'Add ABI B13 Infrared Day/Night integration test products'
```

Comment:
Note there will be an update coming after merging this PR with a minor patch that will make all tiny sector test products Test-SENSOR-VAR-Day-Night/Day-Only/Night-Only, makes it easier to track which product is using which channel when evaluating what to use for a given dataset,  and to tell what each tiny sector test is actually running